src/main/java/com/hotels/housekeeping/conf/Housekeeping.java

```diff
@@ -15,20 +15,25 @@
  */
 package com.hotels.housekeeping.conf;
 
+import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 
 import org.joda.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 @ConfigurationProperties(prefix = "housekeeping")
 public class Housekeeping {
+  private final static Logger log = LoggerFactory.getLogger(Housekeeping.class);
```

Comment:
Looks like this logger isn't actually being used in this class? My guess is you added it to do some debug but have removed the calls to it?

Reply:
I'll remove this